src/EnergyPlus/UnitarySystem.cc

```diff
@@ -10945,7 +10972,8 @@ namespace UnitarySystems {
 
         Real64 FanPartLoadRatio = PartLoadRatio;
         if (this->m_SimASHRAEModel) FanPartLoadRatio = this->FanPartLoadRatio;
-        int SpeedNum = max(this->m_CoolingSpeedNum, this->m_HeatingSpeedNum);
+        if (this->m_EconoPartLoadRatio > 0) FanPartLoadRatio = this->m_EconoPartLoadRatio;
+        int SpeedNum = max(this->m_CoolingSpeedNum, this->m_HeatingSpeedNum, this->m_EconoSpeedNum);
```

Comment:
When `EconomizerFirst` is used, the `FanPartLoadRatio` is not based on the compressor part load ratio but based on the "economizer part load ratio" since the economizer operation dictates the system flow rate.